### ISSUE TITLE
Fixated  interests with abnormal intensity

### ISSUE THREAD

**Issue by bcoleman96** [obophenotype/human-phenotype-ontology#7277]
**Preferred term label:**
Fixated  interests with abnormal intensity

**Synonyms**

**Definition (free text, please give PubMed ID)**
From DSM-V, pg 50

**Parent term (use hpo.jax.org/app)**
Restricted interests (See issue #7275)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0002-4422-1708

**Comment by pnrobinson** [comment]
The definition should be comprehensible to experts and non-specialists. Please enter the suggested DSM-V definition (if we use it, then we should cite the ISBN of DSM-V). It would also be good to provide a PMID if there is a review article that touches on the topic.
In the DSM-V description of autism, we see this
```
Highly restricted, fixated interests that are abnormal in intensity or focus (e.g, strong attachment to or preoccupation with unusual objects, excessively circumscribed or perseverative interest).
```
Should this term also include "focus" or should that be a separate term. Also, providing an example of two (as a comment) is useful. Thanks! @bcoleman96
